### PR TITLE
Updated Chef Workstation, Inspec and Push Job

### PR DESCRIPTION
**Pull request by sootysec**
### Description

Updated to latest version of Chef Workstation, Inspec and Push Job

### Issues Resolved

Updated to latest version of Chef Workstation, Inspec and Push Job

### Check List

- [x ] New functionality includes tests
- [x ] All tests pass
- [x ] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco>
- [ x] PR title is a worthy inclusion in the CHANGELOG

### CODE REVIEW

File: push-jobs/push-jobs.nuspec [hunk 1]
```diff
@@ -19,7 +19,7 @@
     <projectSourceUrl>https://github.com/chef/chef-push</projectSourceUrl>
     <projectUrl>https://github.com/chef/chef-push</projectUrl>
     <releaseNotes>[Release Notes](https://docs.chef.io/release_notes_push_jobs.html)</releaseNotes>
-    <version>2.5.0</version>
+    <version>2.5.06/version>
```
Comment (tas50) [inline push-jobs/push-jobs.nuspec]: these versions don't match. I think you left the 0 by accident here

Comment (tyler-ball) [inline push-jobs/push-jobs.nuspec]:
```suggestion
    <version>2.5.6/version>
```

File: push-jobs/push-jobs.nuspec [hunk 2]
```diff
@@ -19,7 +19,7 @@
     <projectSourceUrl>https://github.com/chef/chef-push</projectSourceUrl>
     <projectUrl>https://github.com/chef/chef-push</projectUrl>
     <releaseNotes>[Release Notes](https://docs.chef.io/release_notes_push_jobs.html)</releaseNotes>
-    <version>2.5.0</version>
+    <version>2.5.6/version>
```
Comment (tyler-ball) [inline push-jobs/push-jobs.nuspec]:
```suggestion
    <version>2.5.6</version>
```

**Comment by sootysec** [comment]
Thanks for the correction. My mistake because I manually edited the file again. Is there anything I need to do to this request now question?

**Review by tyler-ball — APPROVED** [review]

**Pull request merged**